ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallOptions.kt

```diff
@@ -26,19 +26,23 @@ internal sealed class OfferingSelection {
 class PaywallOptions(builder: Builder) {
 
     internal val offeringSelection: OfferingSelection
-    val shouldDisplayDismissButton: Boolean
+    private val shouldDisplayDismissButton: Boolean
```

Comment:
This is a small API change... But I think it's probably ok since we're still on beta. We haven't implemented this yet as well.